packages/core/core/src/BundleGraph.js

```diff
@@ -0,0 +1,140 @@
+// @flow
+import type {
+  Asset,
+  Bundle,
+  BundleGroup,
+  GraphTraversalCallback
+} from '@parcel/types';
+import AssetGraph from './AssetGraph';
+
+const getBundleGroupId = (bundleGroup: BundleGroup) =>
+  'bundle_group:' + bundleGroup.dependency.id;
+
+export default class BundleGraph extends AssetGraph {
```

Comment:
Meant to make this extend from `Graph` instead of `AssetGraph` obviously, but `AssetGraph` has the nice `dumpGraphViz` function. Will move that to `Graph` later.